Newsletter2Go/Export/Api/Data/ResponseInterface.php

```diff
@@ -2,58 +2,56 @@
 
 namespace Newsletter2Go\Export\Api\Data;
 
+/**
+ * @api
+ */
 interface ResponseInterface
 {
-
     /**
-     * @api
      * @return boolean
      */
     public function isSuccess();
 
     /**
-     * @api
      * @param boolean $success
-     * @return null
+     *
+     * @return $this
      */
     public function setSuccess($success);
 
     /**
-     * @api
      * @return string
      */
     public function getMessage();
 
     /**
-     * @api
      * @param string $message
-     * @return null
+     *
+     * @return $this
      */
     public function setMessage($message);
 
     /**
-     * @api
      * @return string
      */
-    public function getErrorcode();
+    public function getErrorCode();
```

Comment:
This change (and the one below) might be wrong... I suspect the connector might expected the key to be `"errorcode"` not `"errorCode"`.